dpdata/lammps/dump.py

```diff
@@ -5,6 +5,9 @@
 lib_path = os.path.dirname(os.path.realpath(__file__))
 sys.path.append(lib_path)
 import lmp
+import warnings
+warnings.simplefilter('once', UserWarning)
```

Comment:
You can define a class derivated from `UserWarning`, i.e.
```py
class SomeWarning(UserWarning):
    pass
```
and throw warnings with that class
```py
warnings.warn('some warning', category=SomeWarning)
```

Reply:
Added.